subsys/bluetooth/shell/bt.c

```diff
@@ -697,6 +697,35 @@ static int cmd_name(const struct shell *sh, size_t argc, char *argv[])
 	return 0;
 }
 
+static int cmd_appearance(const struct shell *sh, size_t argc, char *argv[])
+{
+	if (argc == 1) {
+		shell_print(sh, "Bluetooth Appearance: 0x%04x", bt_get_appearance());
+		return 0;
+	}
+
+	if (argc == 2) {
+		int err;
+		uint16_t new_appearance;
+
+		if (strlen(argv[1]) != 6 || strncmp(argv[1], "0x", 2) || !hex2bin(
```

Comment:
Here I'd create a helper variable called e.g. `const char *val = argv[1];` instead of having the explicitly access it by the array index every time.